frontend/app/controllers/spree/addresses_controller.rb

```diff
@@ -11,10 +11,10 @@ def index
     def create
       @address = spree_current_user.addresses.build(address_params)
       if @address.save
-        flash[:notice] = Spree.t(:successfully_created, scope: :address_book)
-        redirect_to :index
+        flash[:notice] = I18n.t(:successfully_created, scope: :address_book)
+        redirect_to account_path
```

Comment:
@KirillKayumov @pmilewski 

```suggestion
        redirect_to spree.account_path
```